README.md

```diff
@@ -1,244 +1,90 @@
-
-For implicit initialization from environment variables and configuration files as described above, this means that whichever initialization method is used first populates the cache -
-thus, when using the other method, a cache hit happens and no new instance is created. This can lead to surprising misconfigurations:
+A variety of widely-used data science tools are building on `fsspec` to access remote storage resources and can thus work with lakeFS data lakes directly through `lakefs-spec`:
 
 ```python
-import os
-from lakefs_spec import LakeFSFileSystem
-
-# set envvars
-os.environ["LAKEFS_HOST"] = "localhost:8000"
-os.environ["LAKEFS_USERNAME"] = "username"
-os.environ["LAKEFS_PASSWORD"] = "password"
-
-# creates a cache entry for the bare instance
-fs = LakeFSFileSystem()
-
-# ~/.lakectl.yaml
-#  server:
-#    endpoint_url: http://example-host
-
-# this time, try to read in the default lakectl config, with http://example-host set as host.
-fs = LakeFSFileSystem()
-print(fs.client._api.configuration.host) # <- prints localhost:8000!
-```
-
-The best way to avoid this is to commit to only using either environment variables or `lakectl` configuration files.
-If you do have to mix both methods, you can clear the instance cache like so:
-
-```python
-from lakefs_spec import LakeFSFileSystem
+# Pandas -- see https://pandas.pydata.org/docs/user_guide/io.html#reading-writing-remote-files
+import pandas as pd
 
-LakeFSFileSystem.clear_instance_cache()
-```
+data = pd.read_parquet("lakefs://quickstart/main/lakes.parquet")
+print(data.head())
 
-## Reproducibility through data versioning with lakeFS and lakeFS-spec
 
-Here we briefly show an example how data versioning for the reproducibility of machine learning experiments can be achieved using lakeFS-spec. We do this via python-like pseudocode.
+# Polars -- see https://pola-rs.github.io/polars/user-guide/io/cloud-storage/
+import polars as pl
 
-First, we ingest the data into our versioning system.
+data = pl.read_parquet("lakefs://quickstart/main/lakes.parquet")
+print(data.head())
 
-```python
 
-# Data ingestion
-import pandas as pd
-from lakefs_spec.client_helpers import commit
+# DuckDB -- see https://duckdb.org/docs/guides/python/filesystems.html
+import duckdb
+import fsspec
 
-raw = pd.read_csv('local-path-to.csv')
-raw.to_csv('lakefs://<lakeFS-uri')
-commit('raw data ingestion')
+duckdb.register_filesystem(fsspec.filesystem("lakefs"))
+res = duckdb.read_parquet("lakefs://quickstart/main/lakes.parquet")
+res.show()
 ```
 
-This commit function creates a commit with a unique SHA which you can get from the lakeFS user interface or withthe `lakefs_spec.client_helpers.get_tags` function.
+For a more comprehensive overview of third-party tools integrating with `fsspec` (and therefore by extension `lakefs-spec`), see the [user guide](https://lakefs-spec.org/latest/guides/overview/) in the documentation.
```

Comment:
Good, but hold off on this one until the documentation actually contains the user guide.

Reply:
Changed in [35ff4a6](https://github.com/appliedAI-Initiative/lakefs-spec/commit/35ff4a6d0ec34917ee03a911c3fe5a67d435a1d3)